kafka_utils/kafka_consumer_manager/commands/offset_get.py

```diff
@@ -148,3 +150,16 @@ def print_output(cls, consumer_offsets_metadata, watermark_filter):
                             current=metadata_tuple.current
                         )
                     )
+                if watermark_filter == "all" or watermark_filter == "distance":
+                    if metadata_tuple.highmark > 0:
+                        per_behind = float(metadata_tuple.highmark - metadata_tuple.current) / \
```

Comment:
Would this be the ratio behind? For the percentage you need to multiply by 100.

Also, do we need the percentage behind?